CHANGES.rst

```diff
@@ -7,6 +7,9 @@ Unreleased
 **Added**
 
 - :class:`.UserSubreddit` for the ``subreddit`` attribute of :class:`.Redditor`.
+- :meth:`.Redditor.trust` to add a user to the trusted list.
+- :meth:`.Redditor.distrust` to remove a user from the trusted list.
+- :meth:`.User.trusted` to retrieve a ``RedditorList`` of trusted users.
```

Comment:
Kinda nitpicky, you think it would be appropriate for this to be before the other two entries, like at line 10?